airflow/security/kerberos.py

```diff
@@ -55,8 +57,8 @@ def renew_from_kt():
     if subp.returncode != 0:
         log.error("Couldn't reinit from keytab! `kinit' exited with %s.\n%s\n%s" % (
             subp.returncode,
-            b"\n".join(subp.stdout.readlines()),
-            b"\n".join(subp.stderr.readlines())))
+            "\n".join(subp.stdout.readlines()),
+            "\n".join(subp.stderr.readlines())))
```

Comment:
Why this change?

Reply:
@ashb , 
also b"\n".join(subp.stdout.readlines()) caused error on 3.5, it was not catched because wasn't covered with tests

error in 3.5 -
` b"\n".join(subp.stderr.readlines())))

TypeError: sequence item 0: expected a bytes-like object, str found`

Reply:
Do we still need these lines? https://github.com/apache/incubator-airflow/blob/master/airflow/bin/airflow#L26-L28 - I don't know :/ i fix only bug, what args from cli not passed to kerberos.run, but they are defined in cli doc